src/main/java/mate/academy/springboot/datajpa/repository/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    List<Product> findAllByPriceBetween(BigDecimal from, BigDecimal to);
+
+    List<Product> findAllByCategoryId(Long categories);
```

Comment:
Plurals are used for Collections, Long with plural naming looks confusing
```suggestion
    List<Product> findAllByCategoryId(Long categoryId);
```

Reply:
Yes, in the first version I wanted to use List<Long> there